src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -286,6 +294,26 @@ public async Task VaultTimeoutAsync(bool promptOptions = true, int? newTimeout =
             }
         }
 
+        public async Task AppCenterReportingAsync()
+        {
+#if !FDROID && !DEBUG
+            var options = _appcenterActions.Select(s => s == _appCenterReportsDisplayValue ? $"✓ {s}" : s).ToArray();
```

Comment:
I don't see much value on having the allocation for _appcenterActions, I'd just construct the array in situ avoiding the Select and the conversion from List to the Array, even more with just two options. And also I'd extract a method to compose the string with the tick and reuse it on the other methods on the ViewModel:
```c#
var options = new [] 
{  
     CreateSelectableOption(AppResources.Yes, _appCenterReportsEnabled),
     CreateSelectableOption(AppResources.No, !_appCenterReportsEnabled),
};

string CreateSelectableOption(string option, bool selected) => selected ? $"✓ {option}" : option;
```